doc/distrib/NodeHelpFiles/DSRevitNodesUI.RoofTypes.md

```diff
@@ -1,5 +1,7 @@
 ## In Depth
-Allows a selection of a single roof type from all of the roof types available in the current document (file).
+`Roof Types` allows for a selection of a single roof type from all the `Roof Types`` available in the current document (file).
```

Comment:
Can remove the ` from the second "roof types" ("from all the Roof Types available in the current document") because it's ok to translate (it's not a node name in that context)